src/server/command/worldedit/blueprint.zig

```diff
@@ -112,41 +107,35 @@ fn ensureBlueprintExtension(allocator: NeverFailingAllocator, fileName: []const
 	}
 }
 
-fn blueprintDelete(args: List([]const u8), source: *User) !void {
-	if(args.items.len < 2) {
+fn blueprintDelete(args: []const []const u8, source: *User) !void {
+	if(args.len < 2) {
 		source.sendMessage("#ff0000**/blueprint delete** requires FILENAME argument.", .{});
 		return;
 	}
-	if(args.items.len >= 3) {
+	if(args.len >= 3) {
 		source.sendMessage("#ff0000Too many arguments for **/blueprint delete**. Expected 1 argument, FILENAME.", .{});
 		return;
 	}
 
-	const fileName: []const u8 = ensureBlueprintExtension(main.stackAllocator, args.items[1]);
+	const fileName: []const u8 = ensureBlueprintExtension(main.stackAllocator, args[1]);
 	defer main.stackAllocator.free(fileName);
 
-	var cwd = std.fs.cwd();
-
-	_ = cwd.makeDir("blueprints") catch null;
-
-	var blueprintsDir = try cwd.openDir("blueprints", .{});
+	std.fs.cwd().makeDir("blueprints") catch {};
+	var blueprintsDir = try openDir("blueprints");
 	defer blueprintsDir.close();
 
-	try blueprintsDir.deleteFile(fileName);
+	try blueprintsDir.dir.deleteFile(fileName);
 
 	std.log.info("Deleted blueprint file: {s}", .{fileName});
 	source.sendMessage("#ff0000Deleted blueprint file: {s}", .{fileName});
 }
 
-fn blueprintList(_: List([]const u8), source: *User) !void {
-	var cwd = std.fs.cwd();
-
-	_ = cwd.makeDir("blueprints") catch null;
-
-	var blueprintsDir = try cwd.openDir("blueprints", .{.iterate = true});
+fn blueprintList(source: *User) !void {
+	std.fs.cwd().makeDir("blueprints") catch {};
+	var blueprintsDir = try openDir("blueprints");
```

Comment:
Use makeOpenPath instead of makeDir+openDir

Reply:
Also in order to iterate it, you need to set the iterate option, which is not possible with main.files.